pos_longpolling/static/src/js/pos_longpolling.js

```diff
@@ -294,6 +291,9 @@ odoo.define('pos_longpolling', function(require){
         start: function(){
             this._super();
             var self = this;
+            if (!this.pos.config.autostart_longpolling) {
+                $('.serv_primary').remove();
+            }
```

Comment:
Don't remove and make it gray instead. 
Anyway it must be handled in pos_longpolling